tests/openassetio-core-c/private/managerApi/MockManagerInterfaceSuite.hpp

```diff
@@ -22,47 +22,47 @@ namespace test {
  * `mockManagerInterfaceSuite`) will delegate to.
  */
 struct MockCManagerInterfaceImpl {
-  MAKE_MOCK1(dtor, void(oa_managerAPI_CManagerInterface_h));
+  MAKE_MOCK1(dtor, void(oa_managerApi_CManagerInterface_h));
 
   MAKE_MOCK3(identifier,
-             oa_ErrorCode(oa_StringView *, oa_StringView *, oa_managerAPI_CManagerInterface_h));
+             oa_ErrorCode(oa_StringView *, oa_StringView *, oa_managerApi_CManagerInterface_h));
 
   MAKE_MOCK3(displayName,
-             oa_ErrorCode(oa_StringView *, oa_StringView *, oa_managerAPI_CManagerInterface_h));
+             oa_ErrorCode(oa_StringView *, oa_StringView *, oa_managerApi_CManagerInterface_h));
 
   MAKE_MOCK3(info, oa_ErrorCode(oa_StringView *, oa_InfoDictionary_h,
-                                oa_managerAPI_CManagerInterface_h));
+                                oa_managerApi_CManagerInterface_h));
 };
 
 /**
  * Our simulated manager plugin's opaque handle unpacks to an instance
  * of the mock class.
  */
 using MockCManagerInterfaceHandleConverter =
-    openassetio::handles::Converter<MockCManagerInterfaceImpl, oa_managerAPI_CManagerInterface_h>;
+    openassetio::handles::Converter<MockCManagerInterfaceImpl, oa_managerApi_CManagerInterface_h>;
 
 /**
  * Get a ManagerInterface C API function pointer suite that assumes the
  * provided `handle` is a `MockCAPI` instance.
```

Comment:
[consider][minor] A mistake in the docstring, `MockCAPI` should be `MockCManagerInterfaceImpl`.